programming/getting_code_on_the_robot.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: page
+title: Getting Code on the Robot
+---
+
+Getting Code on the Robot
+======================
+
+Getting your code on to the robot is quite simple.
+You will need to put your code on a USB drive
+Which will need to be formatted with either FAT32, exFAT, NTFS or ext2/3/4.
+Upon plugging in the drive or starting up, the robot will run the `robot.py` file found in the `robot.zip` in the root of the drive.
+
+## Windows
+1. Open your code in File Explorer
+2. Select all of your code files (<kbd>Control</kbd><kbd>A</kbd> to select all files)
+3. Right-click the files
+4. Click Compress to Zip file
+5. Name the newly created file `robot` (or `robot.zip` if you have file extentions shown)
+
+![Screenshot for Windows 11]({{ site.baseurl }}/images/content/kit/zip_win11.png)
+
+If you are on Windows 10 or older, you will need to use Send to &rarr; Compressed (zipped) folder
```

Comment:
Worth noting that less than 5% of users are on Win 11 atm.

Reply:
```suggestion
If you are on Windows 10 or older, you will need to use Send to &rarr; Compressed (zipped) folder.
```

Reply:
abd8324